tests/integration_test.py

```diff
@@ -1370,9 +1384,63 @@ def runTest(self):
 
 
 #################
-# BUILDER TESTS #
+# VOLUMES TESTS #
 #################
 
+@requires_api_version('1.21')
+class TestVolumes(BaseTestCase):
+    def test_create_volume(self):
+        name = 'perfectcherryblossom'
+        result = self.client.create_volume(name)
+        self.tmp_volumes.append(name)
```

Comment:
Minor, but if the tests crash or are killed between line 1394 and 1395 the volume won't be cleaned up. Same problem if the client call creates the volume, but then hits a timeout.

I think this needs to be:

```python
name = 'perfectcherryblossom'
self.tmp_volumes.append(name)
result = self.client.create_volume(name)
```

Same for the tests below